### PR TITLE
fixes for hiv STA tutorial

### PR DESCRIPTION
**Pull request by azazellochg**

### CODE REVIEW

File: tomo/templates/tutorial_STA_with_reliontomo_HIV.json.template
Comment (Vilax) [inline tomo/templates/tutorial_STA_with_reliontomo_HIV.json.template:1]: I do not understand why setting twoSurfaces as false: This dataset (HIV) present fiducials at different planes

Here an example
image

**Review by Vilax — COMMENTED** [review]
Thanks for checking the HIV tutorial. I only have a question about the fiducials

File: tomo/templates/tutorial_STA_with_reliontomo_HIV.json.template
Comment (azazellochg) [reply tomo/templates/tutorial_STA_with_reliontomo_HIV.json.template:1]: Hi @Vilax, that's not what this option means. It is only used for plastic section tomography, where the gold is physically deposited on both section surfaces. In cryo-ET, the gold is not arranged into planes but randomly inside the volume. You can check IMOD docs.

Comment (Vilax) [reply tomo/templates/tutorial_STA_with_reliontomo_HIV.json.template:1]: You convince me :)

**Review by Vilax — APPROVED** [review]

**Pull request merged**